pkg/collector/python/util.go

```diff
@@ -79,3 +81,15 @@ func GetSubprocessOutput(argv **C.char, cStdout **C.char, cStderr **C.char, cRet
 	*cStderr = TrackedCString(string(outputErr))
 	*cRetCode = C.int(retCode)
 }
+
+var obfuscator = obfuscate.NewObfuscator(nil)
+
+//export ObfuscateSql
+func ObfuscateSql(rawQuery *C.char) *C.char {
+	s := C.GoString(rawQuery)
+	obfuscatedQuery, err := obfuscator.ObfuscateSQLString(s)
+	if err != nil {
+		return TrackedCString(fmt.Sprint("ERROR: %v", err))
```

Comment:
We should return `nil` here and check the value in C. That way we can set the correct error flag in python

Reply:
Updated